hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/SCMPipelineManager.java

```diff
@@ -153,6 +153,8 @@ protected void initializePipelineState() throws IOException {
           .newBuilder(HddsProtos.Pipeline.PARSER.parseFrom(entry.getValue()));
       Pipeline pipeline = Pipeline.getFromProtobuf(pipelineBuilder.setState(
           HddsProtos.PipelineState.PIPELINE_ALLOCATED).build());
+      // When SCM is restarted, set Creation time with current time.
+      pipeline.setCreationTimestamp(Instant.now());
```

Comment:
Nit: this should go after `Preconditions.checkNotNull` in the next line, otherwise the check becomes unnecessary (will never fail).  The result is the same anyway (NPE).